keyboards/wiredin/protagonist/pro_s/keyboard.json

```diff
@@ -0,0 +1,769 @@
+{
+    "keyboard_name": "Protagonist PRO-S",
+    "manufacturer": "WiredIn",
+    "maintainer": "Gondolindrim",
+    "usb": {
+        "vid": "0x5749",
+        "pid": "0x5053",
+        "device_version": "0.0.1"
+    }, 
+  
+    "matrix_pins": {
+        "cols": ["A3" , "A2" , "A1" , "B14", "B12", "A10", "A8" , "B4" , "B3" , "A15", "C15", "C14", "C13", "B8"],
+        "rows": ["A4" , "A7" , "B0" , "B5" , "B9" , "A0" , "B10"]
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey":  true,
+        "extrakey":  true,
+        "rgblight":  true,
+        "backlight": true,
+        "nkro":      true
+    },
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "backlight": {
+        "pin": "B1",
+        "levels": 20,
+        "as_caps_lock": true
+    },
+    "rgblight": {
+        "led_count": 22,
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        }
+    },
+    "ws2812": {
+        "pin": "B15",
+        "driver": "pwm"
+    },
+    "processor": "STM32F411",
+    "bootloader": "stm32-dfu",
+    "layouts": {
```

Comment:
```suggestion
    "community_layouts": ["60_ansi_tsangan", "60_ansi_tsangan_split_bs_rshift", "60_ansi_wkl", "60_ansi_wkl_split_bs_rshift", "60_hhkb", "60_iso_tsangan", "60_iso_tsangan_split_bs_rshift", "60_iso_wkl", "60_iso_wkl_split_bs_rshift"],
    "layouts": {
```

Reply:
Done in [ee1d48a](https://github.com/qmk/qmk_firmware/pull/22918/commits/ee1d48abe591950db3096628d516857064020446)